### ISSUE TITLE
About the padding_policy

### ISSUE THREAD

**Issue by J-Wu97** [sunxm2357/AdaShare#8]
Thanks for release the codes!

When reading the codes in deeplab_resnet.py :
`skip_layer = sum(self.layers) - num_train_layers
            if cuda_device != -1:
                padding = torch.ones(skip_layer, 2).to(cuda_device)
            else:
                padding = torch.ones(skip_layer, 2)
            padding[:, 1] = 0
            padding_policys = []
            feats = []
            for t_id in range(self.num_tasks):
                padding_policy = torch.cat((padding.float(), self.policys[t_id][-num_train_layers:].float()), dim=0)
                padding_policys.append(padding_policy)

                feats.append(self.backbone(img, padding_policy))
        else:
            feats = [self.backbone(img)] * self.num_tasks`
I wonder if the **padding policy** is to achieve the purpose of curriculum strategy.
Looking forward to your reply,
Thank you!

**Comment by sunxm2357** [comment]
Thanks for your interest!

The padding policy is to achieve curriculum learning. At the beginning of training, only the last several layers are trained so that we use padding policy.

**Comment by J-Wu97** [comment]
> Thanks for your interest!
> 
> The padding policy is to achieve curriculum learning. At the beginning of training, only the last several layers are trained so that we use padding policy.

Thanks for your reply.

I have another question about the 'yaml' file. There are several similar yaml files, such as nyu_v2_task.yaml and nyu_v2_task_test.yaml. I want to know if I use the former one in the training and retrain phases and the latter one in the testing phase.

**Comment by J-Wu97** [comment]
Thanks for your help.
